### PR TITLE
Don't use the local storage object

### PR DESCRIPTION
**Pull request by poncovka**
If the local storage object is not available, don't set it up. It means that
Anaconda uses the DBus Storage module and it is enough to set up the extra
arguments of the Bootloader module.

### CODE REVIEW

**Comment by poncovka** [comment]
This change doesn't depend on anything else, so it is safe to merge this PR and release a new version.

**Comment by poncovka** [comment]
Hi, @daveyoung and @ryncsn, we would like to switch to the DBus Storage module this week and the kdump addon will fail during the installation without this patch. The change will be in Fedora 32 and Rawhide.

File: com_redhat_kdump/ks/kdump.py [hunk 1]
```diff
@@ -83,10 +87,18 @@ def setup(self, storage, ksdata, payload):
...
-            new_args.append(' crashkernel=%s' % self.reserveMB)
+            new_args.append('crashkernel=%s' % self.reserveMB)
+
+        if self.enablefadump and os.path.exists(FADUMP_CAPABLE_FILE):
+            new_args.append('fadump=on')
 
         bootloader_proxy.SetExtraArguments(new_args)
 
+        # If the local storage object is not available, skip.
+        # FIXME: This is a temporary workaround.
```
Comment (ryncsn) [inline com_redhat_kdump/ks/kdump.py]: Hi, about this FIXME, if local storage object is not available (Anaconda uses the DBus Storage module), then the below code, which remove duplicated boot_args and append proper boot_args,  do we need to implement that again for the DBus Storage module?

Comment (poncovka) [reply com_redhat_kdump/ks/kdump.py]: No, it is enough to set up the extra arguments above. The storage module will collect all arguments right before the bootloader installation, so it is safe to change them at this point. I have tested both versions and they worked fine.

Comment (ryncsn) [reply com_redhat_kdump/ks/kdump.py]: Thanks, but why say it's a temporary workaround then?

Comment (poncovka) [reply com_redhat_kdump/ks/kdump.py]: Once Anaconda uses the DBus Storage module, we can completely remove the code for the local storage object, so it's only a temporary solution. I will open another PR later. We plan to release a new Anaconda tomorrow, could you please do a new release with this patch?

Comment (ryncsn) [reply com_redhat_kdump/ks/kdump.py]: Thanks, looks good to me, merged and will make a release now.

Comment (poncovka) [reply com_redhat_kdump/ks/kdump.py]: Thank you!

**Pull request merged**